Remove viewpunch

allvei opened the pull request:
Remove DEVONLY flag from "sv_surpress_viewpunch" and change the default to 1.
Apply cvar across codebase to cover all fields.
Fixes p4sstime/p4ss#199

allvei committed:
Remove camera recoil
Fixes p4sstime/p4ss#199

allvei committed:
Merge branch 'development' into allvei/issue199

ardaboyroo reviewed (changes requested):
Currently the convar conveyes that every viewpunch (falldamage, weapon recoil, weapon damage) will be disabled, however that is not the case. Instead only the weapon recoil viewpunch is disabled.

I recommend changing the convar name to better represent its purpose, or to include the other viewpunch sources as well.

allvei commented:
> Currently the convar conveyes that every viewpunch (falldamage, weapon recoil, weapon damage) will be disabled, however that is not the case. Instead only the weapon recoil viewpunch is disabled.
> 
> I recommend changing the convar name to better represent its purpose, or to include the other viewpunch sources as well.

Are you not able to change it yourself? Github supports editing the files directly.on the platform and crwates a new commit.

ardaboyroo commented:
I can, but my comment was more meant to ask which one should be implemented. I'll bring it up with the rest of the devs to see their opinions on this.

allvei committed:
Merge branch 'development' into allvei/issue199

ardaboyroo reviewed (changes requested):
After some discussion, we agreed that the convar should remove the viewpunch for every source (falldamage, weapon damage, etc.)

allvei committed:
Merge branch 'development' into allvei/issue199

allvei committed:
feat: enable sv_suppress_viewpunch by default and remove tf_viewpunch_disable

- Changed sv_suppress_viewpunch default from 0 to 1
- Removed FCVAR_DEVELOPMENTONLY flag from sv_suppress_viewpunch
- Replaced tf_viewpunch_disable with sv_suppress_viewpunch
- Added sv_suppress_viewpunch check to ApplyPunchImpulseX and fall landing effects

ardaboyroo reviewed (approved):
LGTM
